.gitignore

```diff
@@ -63,6 +62,9 @@ instance/
 # Scrapy stuff:
 .scrapy
 
+# Media stuff:
+media/avatars
```

Comment:
Might as well just filter out the entire `media` folder, then use `!` to include specific files. For example:

```gitignore
media/
!media/default.jpg
```

Also how about moving `default.jpg` to `media/avatars`? Seems more appropriate to me.